resources/postgres/postgres.go

```diff
@@ -44,7 +44,7 @@ func (i *Instance) Start(port int, stdout, stderr io.Writer, waitSec int) (err e
 
 	p := exec.Command(pgbin, "start", "-w",
 		"-D", i.datadir,
-		"-o", fmt.Sprintf(`"-p %d"`, i.port))
+		"-o", fmt.Sprintf(`-p %d -k . -h ""`, i.port))
```

Comment:
Let's mention what does -k . -h "" mean

probably mention the link https://www.postgresql.org/docs/current/app-postgres.html